index.ts

```diff
@@ -138,17 +137,13 @@ export class Address extends Bytes {
 }
 
 /** An arbitrary size integer represented as an array of 64 bit values. */
```

Comment:
The comment also needs to be updated. I wonder why we went with `U64Array` at first...

Reply:
Done. I've updated the PR, removing `U64Array` and moving the `toString()` and `toHex()` conversions into the `BigInt` class.